docs/index.rst

```diff
@@ -108,6 +108,7 @@ Content
 
     Operators and hooks <operators-and-hooks-ref>
     CLI <cli-ref>
+    CLI migration guide <cli-migration-guide>
```

Comment:
In the next PR, maybe you could create a dedicated "Airflow 2.0 Migration Guides" section and have all the migration under that, WDYT??

Reply:
Nice. I will do that.